databases/mydb/migrations/03_add_bookmarks.up.sql

```diff
@@ -0,0 +1,49 @@
+------------
+-- Tables --
+------------
+
+DROP TABLE IF EXISTS folders;
+CREATE TABLE folders
+(
+    id         BIGSERIAL PRIMARY KEY,
+    name       VARCHAR(256),
+    user_id    BIGINT REFERENCES users                NOT NULL,
+    created_at TIMESTAMP WITH TIME ZONE DEFAULT now() NOT NULL
+);
+
+DROP TABLE IF EXISTS bookmarks;
+CREATE TABLE bookmarks
+(
+    id          BIGSERIAL PRIMARY KEY,
+    name        VARCHAR(256),
+    user_id     BIGINT REFERENCES users                NOT NULL,
+    source_uid  CHAR(8)                                NOT NULL,
+    source_type VARCHAR(32)                            NOT NULL,
+    data        JSONB,
```

Comment:
please rename `data` to `properties` for system wide consistency.
you had `data` for history where it makes sense as it comes from chronicles but in archive it is `properties`.

Reply:
looks like you changed this in the code but not in the migration ...

Reply:
https://github.com/Bnei-Baruch/archive-my/blob/a6fae372e81c56d62a8a53129b3891f6bc8552c0/databases/mydb/migrations/04_rename_columns.up.sql#L6